meteor/sofie-server/data-access/repositories/mongo/mongo-entity-converter.ts

```diff
@@ -7,12 +7,24 @@ import { Timeline } from '../../../model/entities/timeline'
 import { Identifier } from '../../../model/interfaces/identifier'
 import { AdLibPiece } from '../../../model/entities/ad-lib-piece'
 import { PieceLifeSpan } from '../../../model/enums/piece-life-span'
+import { BasicRundown } from '../../../model/entities/basic-rundown'
 
 export interface MongoIdentifier {
 	_id: string
 	name: string
 }
 
+export interface MongoRundownPlaylist extends Document {
+	_id: string
+	created: number
+	rundownIdsInOrder: string[]
+	externalId: string
+	studioId: string
+	name: string
+	rehearsal: string
+	activationId: string
+}
```

Comment:
At the time of writing, do we need all these attributes? 

I think we should try and only fetch the data we actually need and then add attributes here when we discover that we need them.

Reply:
I have removed the unused attributes for now.